src/test/java/com/epam/prejap/teatrees/IBlockTest.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.prejap.teatrees;
+
+import com.epam.prejap.teatrees.block.Block;
+import com.epam.prejap.teatrees.block.IBlock;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+/**
+ * @author Marta Krawczykowska, Krzysztof Dolanski
+ */
+public class IBlockTest {
+
+
+    @Test
+    public void imageShouldBeTheSameAsGivenGeometricalFigure(){
+
```

Comment:
IMO could be clearer if you used an array or something to show the shape of the block?

Reply:
Done by usage of @DataProvider. Thanks for a hint.